recipes/bigdft/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "bigdft-suite" %}
+{% set version = "1.9.3_dev" %}
+{% set build = 0 %}
+{% set mpi = mpi %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/l_sim/bigdft-suite/-/archive/Prerelease1.9.3/bigdft-suite-Prerelease1.9.3.tar.gz
```

Comment:
The URL should depend on `version` to avoid issues with accidentally deploying a version with the wrong sources.

Also is this a pre-release?

Reply:
> The URL should depend on version to avoid issues with accidentally deploying a version with the wrong sources.

Done. Thanks for the correction.

> Also is this a pre-release?

That is my plan. We will release version 1.9.3 of BigDFT in about a month and a half, and I'd like to have a pre-release available so all developers can try out the conda approach. I believe that my `conda_build_config.yaml` should properly label this as a pre-release, but let me know if there are fixes I should make.